apps/condo/domains/common/utils/helpers.spec.ts

```diff
@@ -1,6 +1,47 @@
-import { getFiltersFromQuery } from '@condo/domains/common/utils/helpers'
+import { getFiltersFromQuery, formatDate, stripPhone } from '@condo/domains/common/utils/helpers'
+import { RU_LOCALE, EN_LOCALE } from '@condo/domains/common/constants/locale'
 
 describe('Helpers property', () => {
+    describe('stripPhone', () => {
+        it('outputs provided phone number in format of +7123456789', () => {
+            expect(stripPhone('+7 (999) 111-22-34')).toBe('+79991112234')
```

Comment:
normalizePhone

Reply:
It's not a good name, because "normalize" is an abstract word and does't describes a final result.
Normalize to what format?
"Strip" describes intent better, — we "strip" (omit) a string.
An example here https://apidock.com/ruby/String/strip

Reply:
Used in `fb03c61c`